js-functions-arrays/functions-arrays.js

```diff
@@ -34,8 +28,8 @@ let myFunctionArrowResult = myFunctionArrow (8,3);
 console.log('result of arrow function', myFunctionArrowResult);
 
 // 1.3. Arrow function, varian 2:
-let myFunctionArrowBoolean = (a, b) => a > b ? a : b;
-let myFunctionArrowResultSimple = myFunctionArrowBoolean (12,4);
+let myFunctionArrowBoolean = (a, b) => {return a > b ? a : b};
```

Comment:
you don't need to use "return" if there is only 1 line with code. If in arrow func 1 line with code, you can omit {} and return